src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -7,6 +7,7 @@ import ToggleSwitch from '../../../components/ToggleSwitch';
 import {colors} from '../../../utils/colors';
 import {profileSettingsDMpermission} from '../../../service/profile';
 import {addDotAndRemoveNewline} from '../../../utils/string/TrimString';
+import { COLORS } from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` from `'../../../utils/theme'` is added, but the old import `{colors} from '../../../utils/colors'` is not removed. If `COLORS` is replacing `colors`, then the old import should be removed to avoid unused imports and potential confusion.
```diff
- import {colors} from '../../../utils/colors';
+ import { COLORS } from '../../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->